lib/travis/api/v3/queries/build.rb

```diff
@@ -36,9 +36,14 @@ def restart(user)
       end
     end
 
-    def priority
+    def priority(user)
```

Comment:
I have couple of concerns with this method

1. It seems to be doing 2 things.
2. Name of the method does not represent completely what it does, lets rename it to something better.